specifyweb/frontend/js_src/lib/localization/query.ts

```diff
@@ -650,28 +650,8 @@ export const queryText = createDictionary({
   importNoReadPermission: {
     'en-us':
       'The query you imported contains tables you do not have read access to:',
-    'es-es': `
```

Comment:
Looks like the merge didn't go well once again. Not sure why this keeps on happening.
Do you want to revisit merge conflict resolution on Monday?

Reply:
Fixed by removing "virus" merge commits